modules/fidelityBidAdapter.js

```diff
@@ -59,11 +71,27 @@ export const spec = {
     }
     return bidResponses;
   },
-  getUserSyncs: function getUserSyncs(syncOptions) {
-    if (syncOptions.iframeEnabled) {
-      return [{
-        type: 'iframe',
-        url: '//' + BIDDER_SERVER + '/delivery/matches.php?type=iframe',
+  getUserSyncs: function getUserSyncs(syncOptions, serverResponses, gdprConsent) {
+    if (syncOptions.iframeEnabled) {
+      var url = '//' + BIDDER_SERVER + '/delivery/matches.php?type=iframe';
+      if (gdprConsent) {
+        var gdpr = 0;
+        var consent_str = '';
+        var consent_given = 0;
+        if (typeof gdprConsent.gdprApplies !== 'undefined') {
```

Comment:
Move this duplicate logic in one function

Reply:
Thanks for the review. We are making necessary changes.

Reply:
@jaiminpanchal27  Changes were made, please review.